### DISCUSSION TITLE
Can't compile without error when trying to do ubl with latest bug fix can some one help

### DISCUSSION THREAD

**Discussion by blackoutlaw** [mriscoc/Ender3V2S1#1278] [Q&A]
i have an SKR mini E3 V3 and i used the special configuration pwy for getting the config files.  the settings i did was ender3v2, SKRME3V3, BLT, UBL, DACAI, T5, Speaker, NP, MPC.  when i tried to compile it errored with the following:

Linking .pio\build\STM32G0B1RE_btt\firmware.elf
c:/users/theon/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/12.3.1/../../../../arm-none-eabi/bin/ld.exe: Marlin/lib/proui\libproui.a(proui_ex.cpp.o): in function `MarlinSettings::store_mesh(signed char)':
(.text._ZN14MarlinSettings10store_meshEa+0x36): undefined reference to `unified_bed_leveling::set_store_from_mesh(float const (&) [9][9], short (&) [9][9])'
c:/users/theon/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/12.3.1/../../../../arm-none-eabi/bin/ld.exe: Marlin/lib/proui\libproui.a(proui_ex.cpp.o): in function `MarlinSettings::load_mesh(signed char, void*)':
(.text._ZN14MarlinSettings9load_meshEaPv+0x60): undefined reference to `unified_bed_leveling::set_mesh_from_store(short const (&) [9][9], float (&) [9][9])'
collect2.exe: error: ld returned 1 exit status
*** [.pio\build\STM32G0B1RE_btt\firmware.elf] Error 1

also when I go into any of the menus I don't get a led menu or case light menu

can anyone help me with this issue

**Comment by SergiuToporjinschi** [comment]
Hi @blackoutlaw,
did you found the problem? i have the same issue

**Reply by heidepiek** [reply]
Hope the files were of some use to you. But I take them off my drive again. They do take up quite a bit of space. If anyone needs them, they can ask for them.

 harrie
